cmd/things/main.go

```diff
@@ -64,6 +64,11 @@ const (
 	streamID = "magistrala.things"
 )
 
+var (
+	authClient   magistrala.AuthServiceClient
+	policyClient magistrala.PolicyServiceClient
+)
```

Comment:
Why we have global varaible for gRPC clients

Reply:
This can be moved inside `main` function
Because anyhow we are passing clients to functions.